test/allowlist_for_publicAPI.json

```diff
@@ -890,7 +890,9 @@
     "addmm",
     "log_softmax",
     "mm",
-    "softmax"
+    "softmax",
+    "SemiStructruedSparseTensor",
+    "to_semi_structured_sparse_tensor"
```

Comment:
Is this a leftover?

Reply:
This is to for `test_public_bindings`

Reply:
You should never need to add stuff here. Just make sure that these functions are properly exposed and documented!